.gitlab-ci.yml

```
@@ -0,0 +1,39 @@
+image: registry.ddbuild.io/images/bazel:5-gbi
+
+variables:
+  KUBERNETES_CPU_REQUEST: 4
+  KUBERNETES_MEMORY_REQUEST: 6Gi
+  KUBERNETES_MEMORY_LIMIT: 12Gi
+  GONOSUMDB: "github.com/DataDog,go.ddbuild.io"
+  GOPROXY: "binaries.ddbuild.io,proxy.golang.org,direct"
+
+before_script:
+  - git config --global url."https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/".insteadOf "https://github.com/DataDog/"
+
+stages:
+  - test
+  - push
+
+build-and-push-images:
+  tags:
+    - "arch:amd64"
```

Could we maybe use arm since it is less used at the moment and less expensive or is it mandatory for this job to run with amd ?
```suggestion
    - "arch:arm64"
```